lib/src/server/service.dart

```diff
@@ -34,6 +37,56 @@ class ServiceMethod<Q, R> {
       this.streamingResponse,
       this.requestDeserializer,
       this.responseSerializer);
+
+  StreamController<Q> createRequestStream(StreamSubscription incoming) =>
+      new StreamController(
```

Comment:
Is `new StreamController<Q>` inferred here?

Reply:
I think so - it works, or at least I don't get the runtime exception anymore. :)

I'll make it explicit.